action_server/build.py

```diff
@@ -125,36 +120,50 @@ def _resume_download(url: str, filename: str | Path, chunk_size: int = 1024):
             stream.write(chunk)
 
 
-def _download_rcc(system: Optional[str] = None, target: Optional[str] = None):
+def _download_rcc():
     """
     Downloads RCC in the place where the action server expects it.
     """
     import stat
     import urllib.request
 
-    if target:
-        rcc_path = Path(target)
+    if sys.platform == "win32":
+        rcc_path = (
+            CURDIR
+            / "src"
+            / "sema4ai"
+            / "action_server"
+            / "bin"
+            / f"rcc-{RCC_VERSION}.exe"
+        )
     else:
-        if sys.platform == "win32":
-            rcc_path = (
-                CURDIR
-                / "src"
-                / "sema4ai"
-                / "action_server"
-                / "bin"
-                / f"rcc-{RCC_VERSION}.exe"
-            )
+        rcc_path = (
+            CURDIR / "src" / "sema4ai" / "action_server" / "bin" / f"rcc-{RCC_VERSION}"
+        )
+
+    machine = platform.machine()
+    is_64 = not machine or "64" in machine
+
+    if sys.platform == "win32":
+        if is_64:
+            relative_path = "/windows64/rcc.exe"
+        else:
+            relative_path = "/windows32/rcc.exe"
```

Comment:
We don't have 32 bit versions on RCC micromamba etc. so this could be dropped.

Reply:
Ok, raising exception in this case to make it clear.